migrate/migrate.go

```diff
@@ -100,6 +112,54 @@ func (m *Migrator) migrateToV0_1_0(id uuid.UUID) error {
 	return nil
 }
 
+// migrateToV0_12_0 migrates a project to the version v0.12.0
+//
+// Steps:
+// - 1. Reset project state recreate initial accounts
+// - 2. Get all accounts for project and update with shifted addresses and removed unused fields
+func (m *Migrator) migrateToV0_12_0(projectID uuid.UUID) error {
```

Comment:
here we reset project, which basically removes all transaction and script executions, we track every execution of those in db so we can recreate the state at anytime, but when we migrate we remove that as we don't need to preserve the state